drivers/acpi/acpi_processor.c

```diff
@@ -880,8 +880,6 @@ int acpi_processor_evaluate_cst(acpi_handle handle, u32 cpu,
 		memcpy(&info->states[++last_index], &cx, sizeof(cx));
 	}
 
-	acpi_handle_info(handle, "Found %d idle states\n", last_index);
-
```

Comment:
did you intend this to be reviewed @ujfalusi ?

Reply:
Oops